src/SelfServe/SelfServeStyles.tsx

```diff
@@ -0,0 +1,20 @@
+import { IButtonStyles, ICommandBarStyles, ISeparatorStyles, IStackTokens } from "office-ui-fabric-react";
+import { StyleConstants } from "../Common/Constants";
+
+export const commandBarItemStyles: IButtonStyles = { root: { paddingLeft: 20 } };
+
+export const commandBarStyles: ICommandBarStyles = { root: { paddingLeft: 0 } };
+
+export const containerStackTokens: IStackTokens = { childrenGap: 5, padding: 10 };
+
+export const separatorStyles: Partial<ISeparatorStyles> = {
+  root: {
+    selectors: {
+      "::before": {
```

Comment:
Just curious - why ::before instead of just setting the background or foreground color?

Reply:
the Separator is weird that way. I tried the background and foreground color. When i explored the elements rendered, this was the selector present.